### PR TITLE
feat(mjml): Add Cozy styles & custom components

### PR DESCRIPTION
**Pull request by GoOz**
- Added a `layout.mjml` as a base for work
- Added some classes to get Cozy styled components
- Added `MJHeader` & `MJFooter` custom components

### CODE REVIEW

**Comment by Crash--** [comment]
@GoOz You seems to have a problem with your commits ^^

**Comment by GoOz** [comment]
Yeah. Wrong rebase, sorry.

File: packages/cozy-mjml/src/components/MJDefaults.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const validator = require('mjml-validator')
```
Comment (GoOz) [inline packages/cozy-mjml/src/components/MJDefaults.js]: ❤️

**Comment by nono** [comment]
[(𝐌) [NUA] ETU j'ai des emails personnalisé en fonction de mon offre](https://trello.com/c/vXXMFYl5/1071-%F0%9D%90%8C-nua-etu-jai-des-emails-personnalis%C3%A9-en-fonction-de-mon-offre)

File: packages/cozy-mjml/src/components/MJFooter.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        <mj-section padding="16px 0 8px">
+          <mj-column align="center">
+            <mj-image padding="0" width="32px" src="../src/assets/images/cozy-logo-round.png"></mj-image>
+          </mj-column>
+        </mj-section>`
+    }
+  }
+
+  render() {
+    if (this.getAttribute('locale') === 'fr') {
```
Comment (gregorylegarec) [inline packages/cozy-mjml/src/components/MJFooter.js]: Maybe we could make the code a little bit simpler and avoid repeating markup with something like:
```js
const locales = {
  fr: {
    tagline: "Cozy, votre domicile numérique"  
	// ...
  },
  en: {
    tagline: "Cozy, your smart personal cloud"
    // ...
  } 
}

// ...

class MJFooter extends core.BodyComponent {
  // ...
  render(){
    const lang = this.getAttribute('locale')
    const translations = locales[lang]

    return this.renderMJML(
      `<mj-wrapper padding="0">
       // ...
          <mj-text padding="0" align="center" mj-class="highlight">${translations.tagline}</mj-text>
       // ...
    )
  }
}
```

Comment (gregorylegarec) [reply packages/cozy-mjml/src/components/MJFooter.js]: Ok, @nono did it in the next commits 👍

**Pull request merged**